build-packages/scandipwa-template-plugin-system/error-handlers/possible-plugin-error.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable consistent-return */
+const logger = require('@scandipwa/scandipwa-dev-utils/logger');
+
+/**
+ * This decorator function handles errors if any are thrown from the subject (cb).
+ * The further execution is halted on error. The user is notified.
+ *
+ * @param {function} cb callback to attempt to execute
+ * @param {string} name processable plugin's name
+ * @returns {any} the result of the callback
+ */
+module.exports = (name, cb) => {
+    if (!name) {
+        throw new Error(
+            'The plugin\'s name has not been provided to the possible error handler!'
+        );
+    }
+
+    try {
+        return cb();
+    } catch (err) {
+        const { message = '' } = err;
+
+        // reformat the message for prettier outpit
+        const trimmedMessage = message.replace(/^( |\t)+/gm, '');
+        const trimmedMessageLines = trimmedMessage.split('\n');
+
+        logger.error(
+            `${logger.style.code(name)} module's template plugin has thrown the following error:`,
+            ...trimmedMessageLines
+        );
+
+        process.exit(255);
```

Comment:
The exit code here - why "exit status out of range"? Do we need this exists with different statuses? It is very hard to understand by reading. Consider moving to meaningfull consts (rememer: no magic numbers!).

Reply:
I mixed this up - 1 is the "general error" code, I thought 255. Fixed to 1.

Reply:
It still appears as 255 code, no?